src/data/index_array_type.js

```diff
@@ -1,25 +1,12 @@
 // @flow
 
-const createStructArrayType = require('../util/struct_array');
-
 /**
  * An index array stores Uint16 indicies of vertexes in a corresponding vertex array. We use
  * two kinds of index arrays: arrays storing groups of three indicies, forming triangles; and
  * arrays storing pairs of indicies, forming line segments.
  * @private
  */
-
-function createIndexArrayType(components: number) {
-    return createStructArrayType({
-        members: [{
-            type: 'Uint16',
-            name: 'vertices',
-            components
-        }]
-    });
-}
-
 module.exports = {
-    LineIndexArray: createIndexArrayType(2),
-    TriangleIndexArray: createIndexArrayType(3)
+    LineIndexArray: require('./array_type/line_index'),
```

Comment:
I'm tempted to say these could just be straight `Uint16Array`s, or light wrappers around them.